test/integration/targets/file_lock/inventory

```diff
@@ -1,2 +1,2 @@
 [lockhosts]
-lockhost[00:99] ansible_connection=local
+lockhost[00:99] ansible_connection=local ansible_python_interpreter="{{ ansible_playbook_python }}"
```

Comment:
This is required by the test infrastructure to avoid using interpreter discovery in tests. Using interpreter discovery adds additional overhead and can result in selecting the wrong interpreter for tests.

Reply:
Thanks for the explanation. I thought it was due to oddities around which Python is actually being used when running locally.